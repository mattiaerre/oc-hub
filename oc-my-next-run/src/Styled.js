import styled from 'styled-components';

export const Title = styled.h1`
  font-size: 1.5em;
  text-align: center;
`;

export const Wrapper = styled.section`
  padding: 2em;
  background: floralwhite;
  font-family: monospace;
  border: 1px solid grey;
`;
