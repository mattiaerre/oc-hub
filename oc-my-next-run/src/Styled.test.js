import React from 'react';
import { create } from 'react-test-renderer';
import { matcher, serializer } from 'jest-styled-components';
import { Title, Wrapper } from './Styled';

expect.extend(matcher);
expect.addSnapshotSerializer(serializer);

test('Title', () => {
  const tree = create(<Title>Hello from oc-my-next-run Title</Title>).toJSON();
  expect(tree).toMatchStyledComponentsSnapshot();
});

test('Wrapper', () => {
  const tree = create(<Wrapper>Hello from oc-my-next-run Wrapper</Wrapper>).toJSON();
  expect(tree).toMatchStyledComponentsSnapshot();
});
