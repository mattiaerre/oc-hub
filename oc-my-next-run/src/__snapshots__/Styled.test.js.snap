// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title 1`] = `


.kbYEP {
  font-size: 1.5em;
  text-align: center;
}

<h1
  className="sc-bdVaJa kbYEP"
>
  Hello from oc-my-next-run Title
</h1>
`;

exports[`Wrapper 1`] = `


.bOYEsa {
  padding: 2em;
  background: floralwhite;
  font-family: monospace;
  border: 1px solid grey;
}

<section
  className="sc-bwzfXH bOYEsa"
>
  Hello from oc-my-next-run Wrapper
</section>
`;
